terraform/modules/azure/storage.tf

```diff
@@ -0,0 +1,33 @@
+resource "azurerm_storage_account" "tempdata" {
+  name                              = var.tempdata_storage_account_name
+  resource_group_name               = var.resource_group_name
+  location                          = var.region_name
+  account_replication_type          = var.environment_name != "production" ? "LRS" : "RAGRS"
+  account_tier                      = "Standard"
+  access_tier                       = "Hot"
+  account_kind                      = "StorageV2"
+  min_tls_version                   = "TLS1_2"
+  enable_https_traffic_only         = true
+  public_network_access_enabled     = true
+  default_to_oauth_authentication   = false
+  infrastructure_encryption_enabled = false
+  allow_nested_items_to_be_public   = false
+
+  blob_properties {
+    delete_retention_policy {
+      days = var.tempdata_delete_retention_days
```

Comment:
The value of tempdata_delete_retention_days can be hardcoded since it doesn't change across environments

Reply:
Hardcoded